scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -49,16 +55,22 @@
 
 {% macro filter_form_footer(form, result_size, total_variants, more_variants, page, nvars) %}
 <div class="card-footer text-center">
-  <div class="row">
-    <div class="col-2 d-flex align-items-baseline justify-content-start">
-      <input type="hidden" id="expand_search" name="expand_search">
-      {{ form.hide_dismissed }}
-      {{ form.hide_dismissed.label(class="ml-3") }}
+  <input type="hidden" id="expand_search" name="expand_search">
+  <div class="row d-flex d-flex justify-content-start">
+    <div class="col-3">
+      <div class="form-check d-flex justify-content-start">
+        {{ form.hide_dismissed(class="form-check-input") }}
+        {{ form.hide_dismissed.label(class="form-check-label") }}
+      </div>
+      <div class="form-check d-flex justify-content-start">
+        {{ form.hide_unaffected(class="form-check-input") }}
+        {{ form.hide_unaffected.label(class="form-check-label") }}
+      </div>
     </div>
-    <div class="col-3 offset-3 d-flex justify-content-start">
+    <div class="col-3 d-flex justify-content-end">
       Filter returns {{result_size}} / {{ total_variants }} variants.
     </div>
-    <div class="col-3 offset-1 d-flex justify-content-end">
+    <div class="col-3 d-flex justify-content-start">
```

Comment:
Hm, this looks odd on my local: 
![Screenshot 2022-03-03 at 12 22 59](https://user-images.githubusercontent.com/758570/156555243-f9e6e097-4b68-4116-89e9-a35f00dc4699.png)
Try the other way around with start-end again?